docs/AboutUs.md

```diff
@@ -16,7 +16,8 @@ We currently do not offer any means to contacting us, thank you for your underst
 [[github](https://github.com/JasonLCY-Temp)]
 [[portfolio](team/jasonlcy-temp.md)]
 
-* Role: Scheduling and tracking, CI/CD Pipeline manager, In charge of `Interview`
+* Role: Scheduling and tracking, CI Pipeline manager
+* Responsibilities: In charge of  `Interview`
```

Comment:
I feel like the responsibility can be specified in slightly more detail? For example, create `Interview` object? Might be quite subjective though.

Reply:
Good discussion point for us to cover in the Week 8 meeting, I've added it to the meeting agenda in our meeting notes.